workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -545,7 +545,7 @@ PlayerSettings:
   webGLCompressionFormat: 1
   webGLLinkerTarget: 0
   scriptingDefineSymbols:
-    1: UNITY_POST_PROCESSING_STACK_V2;UNITY_DISABLE_AUTOMATIC_SYSTEM_BOOTSTRAP
+    1: UNITY_POST_PROCESSING_STACK_V2;UNITY_DISABLE_AUTOMATIC_SYSTEM_BOOTSTRAP;IMPROBABLE_WORKERTYPE_UNITYCLIENT
```

Comment:
do we need that extra symbol?

Reply:
They know!

Reply:
Actually the build system added that, but, it shouldn't...
This is literally unset in a `finally` block.. I have no idea how that gets there